### ISSUE TITLE
Support for Parquet v1.5.1-SNAPSHOT

### ISSUE THREAD

**Issue by 0xADD1E** [cldellow/sqlite-parquet-vtable#41]
All recent releases of Arrow/Parquet are on API version 1.5.1-SNAPSHOT (back to at least release 0.11.0, which is the earliest version I can find in Apache's binary repo).

This brings a few incompatibilities which will need working around in this codebase. Has anyone else looked into this?

**Comment by cldellow** [comment]
Maybe have a look at https://github.com/cldellow/sqlite-parquet-vtable/pull/39, where @mharju did some work on it.

It's not quite perfect, as they were trying to get it to build on Mac OS X using dynamic libs. I think it successfully builds for them on OS X, but doesn't on Ubuntu.

I tried to update the Ubuntu build bits (see https://github.com/cldellow/sqlite-parquet-vtable/pull/39#issuecomment-553983872), but am pretty sure I did something wrong with setting the includes directory, and just haven't found the time to dig into it further.
